docs/content/migration/v2-to-v3.md

```diff
@@ -564,7 +666,7 @@ For this reason, we encourage migrating to the new syntax.
 
 By default, the `defaultRuleSyntax` static option is automatically set to `v3`, meaning that the default rule is the new one.
 
-#### New V3 Syntax Notable Changes
+##### New V3 Syntax Notable Changes
```

Comment:
Could you please take into account the following comment https://github.com/traefik/traefik/issues/10690#issuecomment-2145134722